gslab_scons/tests/test_misc.py

```diff
@@ -120,5 +330,4 @@ def test_lyx_scan(self):
 
```

Comment:
MagicMock() is used here because it is more straightforward to deploy than an SCons `node` object (which it is supposed to be mocking). This mock just gives the contents of a test LyX file when its `get_contents()` method is called. I'll add a comment.